Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/DataGrid/DataGridComboBoxColumn.cs

```diff
@@ -0,0 +1,457 @@
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets a <see cref="T:System.Windows.Controls.ComboBox"/> control that is bound to the column's ItemsSource collection.
+        /// </summary>
+        /// <param name="cell">The cell that will contain the generated element.</param>
+        /// <param name="dataItem">The data item represented by the row that contains the intended cell.</param>
+        /// <returns>A new <see cref="T:System.Windows.Controls.ComboBox"/> control that is bound to the column's ItemsSource collection.</returns>
+        protected override FrameworkElement GenerateEditingElement(DataGridCell cell, object dataItem)
+        {
+            var value = dataItem.GetType().GetProperty(Binding.Path.Path).GetValue(dataItem);
+
+            var items = ItemsSource as IEnumerable<object>;
+
+            var selection = DisplayMemberPath != null
+                ? items?.FirstOrDefault(x => x.GetType().GetProperty(Binding.Path.Path).GetValue(x).Equals(value))
+                : items?.FirstOrDefault(x => x.Equals(value));
+
+            var comboBox = new ComboBox
+            {
+                ItemsSource = ItemsSource,
+                DisplayMemberPath = DisplayMemberPath ?? string.Empty,
+                SelectedItem = selection,
+                Margin = new Thickness(0, 0, 0, 0),
+                HorizontalAlignment = HorizontalAlignment.Stretch,
+                VerticalAlignment = VerticalAlignment.Center
+            };
+
+            if (DependencyProperty.UnsetValue != ReadLocalValue(DataGridComboBoxColumn.FontFamilyProperty))
+            {
+                comboBox.FontFamily = this.FontFamily;
+            }
+
+            if (_fontSize.HasValue)
+            {
+                comboBox.FontSize = _fontSize.Value;
+            }
+
+            if (_fontStyle.HasValue)
+            {
+                comboBox.FontStyle = _fontStyle.Value;
+            }
+
+            if (_fontWeight.HasValue)
+            {
+                comboBox.FontWeight = _fontWeight.Value;
+            }
+
+            RefreshForeground(comboBox, (cell != null & cell.OwningRow != null) ? cell.OwningRow.ComputedForeground : null);
+
+            comboBox.SelectionChanged += (sender, args) =>
+            {
+                var item = args.AddedItems.FirstOrDefault();
+                if (item != null)
+                {
+                    var newValue = DisplayMemberPath != null
+                        ? item.GetType().GetProperty(Binding.Path.Path).GetValue(item)
+                        : item;
+
+                    dataItem.GetType().GetProperty(Binding.Path.Path).SetValue(dataItem, newValue);
+                }
+            };
+
+            return comboBox;
+        }
+
+        /// <summary>
+        /// Gets a read-only <see cref="T:System.Windows.Controls.TextBlock"/> element that is bound to the column's <see cref="P:Microsoft.Toolkit.Uwp.UI.Controls.DataGridBoundColumn.Binding"/> property value.
```

Comment:
T:Windows.UI.Xaml.Controls.TextBlock
(Text box column is incorrect too)

Reply:
Will change it.